### ISSUE TITLE
Karma Expenditure on new Skills not calculated correctla

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#87]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/13_

----

```
What steps will reproduce the problem?
1. The character needs to be in career mode
2. You want to spend Karma on a new skill

What is the expected output? What do you see instead?
I would expect the new skill to cost 2 Karma as per description in the Rules. 
It instead costs 4 Karma

What version of the product are you using? On what operating system?
0.0.5.139
Windows 8

Please provide any additional information below.


```

Original issue reported on code.google.com by `nino.big...@gmail.com` on 29 Sep 2014 at 9:45

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Oh and the character doesn't have the Uneducated Quality
```

Original comment by `nino.big...@gmail.com` on 1 Oct 2014 at 12:14

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
I can't seem to replicate this problem... it just takes 2KP on my version...

```

Original comment by `DonovanA...@gmail.com` on 18 Nov 2014 at 10:06

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Here's two Characters, which are both causing the problem. Sorry for taking so 
long.
```

Original comment by `nino.big...@gmail.com` on 9 Dec 2014 at 12:42

Attachments:
* [Cat02.chum5](https://storage.googleapis.com/google-code-attachments/chummer5/issue-13/comment-3/Cat02.chum5)
* [BitShiftCareer.chum5](https://storage.googleapis.com/google-code-attachments/chummer5/issue-13/comment-3/BitShiftCareer.chum5)

**Comment by chummer5a** [comment]
Resolved in 5.156.
